x-pack/docs/en/security/securing-communications/update-tls-certificates.asciidoc

```diff
@@ -0,0 +1,529 @@
+[[update-node-certs]]
+== Updating node security certificates
+If your cluster certificates expired or you have a new Certificate Authority
+(CA), then you'll need to update the security certificates for both the
+transport and HTTP layers. When updating your node security certificates, there
+are two main scenarios that you'll encounter:
+
+* <<update-node-certs-same,Your certificate authority (CA) or key remains the same>>
+* <<update-node-certs-different,Your CA certificate or key changes>>
+
+Regardless of the scenario, {es} monitors the SSL resources for updates
+on a five-second interval. You can just copy the new certificate and key files
```

Comment:
mention the 5 sec is the default value?

Reply:
I'll change this to read, "{es} monitors the SSL resources for updates on a five-second interval, which is the default."